model/common/src/icon4py/model/common/exceptions.py

```diff
@@ -22,3 +24,7 @@ def __init__(self, field_name):
 
 class IconGridError(RuntimeError):
     pass
+
+
+def dummy_exchange(dim: gtx.Dimension, field: gtx.Field) -> None:
```

Comment:
that does not belong to `exceptions.py`: if you want to centralize it then put it somewhere in the `testing` for example `test_utils.py`